dkg-gadget/src/signing_manager/mod.rs

```diff
@@ -73,8 +74,8 @@ where
 			work_manager: WorkManager::<B>::new(
 				logger,
 				clock,
-				MAX_RUNNING_TASKS,
-				MAX_ENQUEUED_TASKS,
+				MAX_RUNNING_TASKS * MAX_SIGNING_SETS_PER_PROPOSAL as usize,
+				MAX_ENQUEUED_TASKS * MAX_SIGNING_SETS_PER_PROPOSAL as usize,
```

Comment:
Seems dangerous given our previous issues with the size of the queue no?

Reply:
The second implementation, which is after this commit, no longer includes this code.